src/auto-update/auto-updater.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
should this be under electron folder?

Reply:
Yes. My bad. Will move this under `electron` folder.

Reply:
Updated. Thanks for catching this.